wiki/Tournaments/TCT/v4/en.md

```diff
@@ -0,0 +1,462 @@
+5. Taiko Convert Tournament v4 will use double-elimination bracket, which means that there will be a loser bracket and winner bracket.
+6. Even in a tournament scene osu! is still a game. Please respect each other to everyone, be friendly, and fair play.
+
+### In-game conditions
+
+#### Qualifiers
+
+1. Every players will be invited at the time to the room.
+2. There will be no warmups, no ban phase, and no pick phase.
+3. In the qualifiers round, each map will be played twice, or you can play once and leave if you think you've set your best score.
+4. There will be a maximum time of **2 minutes** in between maps. The referee will use `!mp start 10` command to start the game.
+5. If a player gets disconnected for some reason in the first 10 seconds of the song, the referee will abort the game and restart it.
+   - If it happens after 10 seconds, the score before the disconnect will be counted.
+6. The referee can penalise teams if any player present bad behaviour in the game room.
+7. Once the match has finished, the room will be completely closed.
+8. If you fail, the score **will still be counted**.
+
+#### Group Stage and Knockouts
+
+1. Every players will be invited at the time to the room.
+2. The referee can wait a maximum of **10 minutes** for each player to attend.
+   - If the player doesn't come, the referee will ask the other team for a 1v2 start. It will depend on the other team if they want to play or not.
+   - Have in mind if there's a 1v1 match you are forced to play unless you want a forfeit.
+3. Captains will decide if they want to pick a warmup map (maximum limit is **3:30**) and order doesn't matter.
+4. Captains will use `!roll` command. Lower roll will start banning then higher roll bans (1 ban each team).
+5. After banning stage is ready, higher roll picks their first map then lower roll picks theirs. Continue until the match is done.
+6. There will be a **2 minutes** time limit for all phases.
+7. If the time limit is exceeded
+   - Ban: The ban is treated as none.
+   - Pick: The right is transferred to your opponent. It does not affect the subsequent order.
+   - Before the start: The referee will use `!mp start 10` to start the game.
+8. If a player gets disconnected for some reason in the first 10 seconds of the song, the referee will abort the game and restart it.
+   - If it happens after 10 seconds, the score before the disconnect will be counted.
+9. The referee can penalise teams if any player present bad behaviour in the game room.
+10. Once the match has finished, the room will be completely closed.
+11. FreeMod maps are force mod so at least 1 team member **has to pick a mod**.
+12. Team members aren't allowed to pick the same mod combination.
+13. If a player fails in a song, and doesn't revive then his score is not counted. In the case he revives before map's end it counts. if every player fails, then the referee is going to make a sum of their scores then comparing.
```

Comment:
```suggestion
13. If a player fails in a song and does not revive, then their score is not counted, but if they do revive it will count. If every player in a team fails, then the sum of their scores will count.
```